CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+* Add compatibility support for arrow >= 1.0.2 on newer Python versions while
```

Comment:
Python 2.7 is not supported :)

Reply:
yes, it's probably going to be the last version to support 3.5 and 3.6

Reply:
Fixed

Reply:
```suggestion

* Add compatibility support for arrow >= 1.0.2 on newer Python versions while
```